### DISCUSSION TITLE
Create flashcards using Toggle functionality

### DISCUSSION THREAD

**Discussion by cahreesti** [reuseman/flashcards-obsidian#104] [Ideas]
Hi! Thank you so much for creating this plugin - this is going to be so helpful for med school.

Is there anyway we can import cards based off of our toggles? I like having a list of all my Question / Answers separated by lines using the native toggle functionality and would also like to be able to import this into anki.

**Comment by reuseman** [comment]
What do you mean by having cards from the native toggle functionality exactly?
